Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -476,6 +477,11 @@ bool ESDevice::LaunchPPCTitle(u64 title_id)
   if (!Core::IsRunningAndStarted())
     return BootstrapPPC();
 
+#ifdef USE_RETRO_ACHIEVEMENTS
+  AchievementManager::GetInstance()->HashGame(m_pending_ppc_boot_content_path,
```

Comment:
`m_pending_ppc_boot_content_path` is a folder, not a file. I don't think your hashing code is set up to handle this.

Reply:
Disabling this path for the duration of this PR while I investigate - I suspect a new hashing algorithm entirely may be necessary for this.